pyproject.toml

```diff
@@ -122,7 +122,6 @@ path = "src/jupytext/version.py"
 # Following config is related to JupyterLab extension
 [tool.hatch.build.targets.sdist]
 artifacts = ["jupyterlab/jupyterlab_jupytext/labextension"]
-exclude = ["demo", "**/.yarn", "**/node_modules"]
```

Comment:
This should not be removed. Maybe you wanted to remove only `"demo"`? We should not include `node_modules` and `.yarn`.

Reply:
`hatch` by default does not include non-git files. That's why you have `artifacts` above. `.yarn` and `node_modules` are in the `.gitignore` so they should not be picked up. I have double checked for both `sdist` and `wheel` packages locally.

Reply:
Oh yes! I completly forgot about this feature of `hatch`. That is true. Cheers for checking it!